### PR TITLE
[Modular] Show prisoner policy warnings to prisoners on spawn.

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

This shows prisoners a warning about self-antag and links them to the policy documents on spawn. 
dreamseeker_nDtWUsFUmJ

## How This Contributes To The Skyrat Roleplay Experience

Policy is a good thing.

## Changelog

no true code changes

### CODE REVIEW

File: modular_skyrat/modules/warning/prisoner_warning.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/datum/job/prisoner/after_spawn(mob/living/carbon/human/H, mob/M)
```
Comment (Zonespace27) [inline modular_skyrat/modules/warning/prisoner_warning.dm:1]: one letter vars bad

File: modular_skyrat/modules/warning/prisoner_warning.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+	. = ..()
+	var/linktopolicy = "https://docs.google.com/document/d/e/2PACX-1vQVFGahOD5A1R5qFhtVAmhxN7xz51JCGw-uMaw_WvhkGcB_M_fdQvr7-VxIYEhh5pZnC77eHvuMSrfB/pub"
+	to_chat(M, (span_doyourjobidiot("As a roundstart/latejoin prisoner, you are not an antagonist. You are allowed to cause moderate amounts of drama and shenanigans, but you should not be security's main focus. Do not be unmanageable.  \nRefer to the Prisoner Section of 'General Player Policy and Standards'!\n\nIf you intend to break out of prison, submit an opposing force application or press F1 to admin help it! \n<a href=\"[linktopolicy]\"> Click here to read the policy! </a>")))
```
Comment (Zonespace27) [inline modular_skyrat/modules/warning/prisoner_warning.dm:4]: why is the span in brackets

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/warning/prisoner_warning.dm (hunk 2 above)
Comment (Iamgoofball) [inline modular_skyrat/modules/warning/prisoner_warning.dm:4]: we have an entire system for showing per-role policy as defined by config, why isn't this being placed in there server-side?

**Comment by Iamgoofball** [comment]
always fill out a changelog for your GBP

**Comment by Iamgoofball** [comment]
Code_8YxEY8QAp0
i recommend utilizing the policy.json instead

**Comment by Zonespace27** [comment]
What goof said.

**Comment by nevimer** [comment]
> What goof said.

The rest of the codebase doesn't do that. It all uses this method for every job.

**Comment by Zonespace27** [comment]
> > What goof said.
> 
> The rest of the codebase doesn't do that. It all uses this method for every job.

It shouldn't and now it doesn't (#14288)

**Pull request closed without merging**